mlfinlab/bet_sizing/targeting.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains functionality for determining bet sizes for investments based on volatility targeting.
+"""
```

Comment:
Please add this function to the `mlfinlab/bet_sizing/__init__.py` file

Reply:
I think I also need to add kelly. Am I write?

Reply:
Yes, that's right